packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -152,11 +153,14 @@ class SelectLun extends Component {
 
   _getOptions = createSelector(
     () => this.props.options,
-    options =>
+    () => this.props.intl.formatMessage,
+    (options, formatMessage) =>
       map(options, (lun, index) => ({
-        label: `LUN ${lun.id}: ${lun.serial} - ${formatSize(+lun.size)} - (${
-          lun.vendor
-        })`,
+        label: `LUN ${lun.id}: ${lun.serial} - ${
+          lun.size !== undefined
```

Comment:
IMHO, hide it if it's `undefined`.

Reply:
`unknown` is better because this case isn't normal.